src/pages/components/CertificateActionsButtons.js

```diff
@@ -21,43 +21,20 @@ export default function CertificateActionsButtons({
   return (
     <Sidebar area="sidebar">
       <PaddedDiv>
-        <Grid
-          areas={[
-            ['div-left', 'div-right'],
-            ['div-double', 'div-double'],
-          ]}
-          rows={['40px', '60px']}
-          columns={['1fr', '1fr']}
-          gap="20px 10px"
-        >
-          <Grid.Panel area="div-left">
-            <SmallButton
-              variant="roundedPronounced"
-              onClick={handleClickSaveDraft}
-            >
-              Save draft
-            </SmallButton>
-          </Grid.Panel>
-          <Grid.Panel area="div-right">
-            <SmallButton
-              variant="roundedPronounced"
-              onClick={handleCancelDraft}
-            >
-              Cancel
-            </SmallButton>
-          </Grid.Panel>
-          <Grid.Panel area="div-double">
-            <LargeButton
-              disabled={loading || !valid}
-              variant="roundedPronounced"
-            >
-              {loading == false && data == null && <Span>Submit</Span>}
-              {loading == false && data != null && <Span>Submitted</Span>}
-              {loading && <AnimatedSpan>...</AnimatedSpan>}
-              {error && <Span>Error</Span>}
-            </LargeButton>
-          </Grid.Panel>
-        </Grid>
+        <SmallButton variant="roundedPronounced" onClick={handleClickSaveDraft}>
+          Save draft
+        </SmallButton>
+
+        <SmallButton variant="roundedPronounced" onClick={handleCancelDraft}>
+          Cancel
+        </SmallButton>
+
+        <LargeButton disabled={loading || !valid} variant="roundedPronounced">
+          {loading == false && data == null && <Span>Submit</Span>}
+          {loading == false && data != null && <Span>Submitted</Span>}
+          {loading && <AnimatedSpan>...</AnimatedSpan>}
+          {error && <Span>Error</Span>}
```

Comment:
nice, finally we are using a hook:), to be fair used it in viewer

Reply:
Not my change

Reply:
credit to @dc-andysign